### PR TITLE
Headgibbing is a config thing

### PR DESCRIPTION
**Pull request by Anewbe**
As discussed in or around the Staff Meeting

### CODE REVIEW

File: code/modules/organs/subtypes/standard.dm [hunk 1]
```diff
@@ -268,6 +268,11 @@
 
 	var/eye_icon_location = 'icons/mob/human_face.dmi'
 
+/obj/item/organ/external/head/Initialize()
+	..()
```
Comment (Neerti) [inline code/modules/organs/subtypes/standard.dm]: Generally ``Initialize()`` needs to return a thing and so usually at the bottom is ``return ..()``, similar to ``Destroy()``.

Comment (Anewbe) [reply code/modules/organs/subtypes/standard.dm]: Righto.

**Comment by Wickedtemp** [comment]
Neato, what's it do?

**Comment by Anewbe** [comment]
If the config is turned on, damage that would give a normal limb can, when applied to the head, give the head, as it was in days long past.

**Comment by Wickedtemp** [comment]
Whats the damage threshold for that? 60?

**Comment by Anewbe** [comment]
🤷‍♂️

**Comment by zekesulastin** [comment]
Oh cool, this crap is back.

**Comment by mistyLuminescence** [comment]
but
why

**Comment by Anewbe** [comment]
Done in an effort to stop people from treating death, and especially armed antags, as a trivial slap on the wrist.
Reworked as a config to allow it to be easily toggled off and on where and when appropriate.

**Comment by zekesulastin** [comment]
Here we go, then.  In b4 some antag gets rekt by an exosuit or stacking burn damage and the one doing the wrecking gets banned for their trouble.

**Comment by Woodratt** [comment]
Far as I'm aware it'll just be a reset to how things basically were before we removed head gibbing. 

As for complaints, if you have 'em, point them at Rando.

**Comment by zekesulastin** [comment]
Yeah, and I was there when we removed it too.  I'm somewhat concerned with all the other stuff that hasn't been posted yet since the notes have no details.

Probably a good thing I missed the meeting where this was all discussed because it probably would not have been polite.

**Comment by mistyLuminescence** [comment]
Isn't this why decapitation exists, though, and the super difficult surgery to fix it? This reads like a bad D&D DM trying to get people to die forever instead of, like, actually playing the round with the character they want to play. 

"You find the patient and they are dead! Very dead!"
"Oof. Can I try and bring them back? I've got a defib and expertise in Medicine."
"No, uh, their head... fell off. It's across the room."
"Alright, let's reattach it and get to work!"
"Oh, uh, their head fell off *real bad*. You'll have to do some really tricky surgery to save them now."
"Let's try the surgery... Nat 20! How'd I do?"
"Actually, I changed my mind, their head just explodes. There's nothing to reattach."

The answer to "antag violence is ineffective" is... which? 
A) "encourage antags to actually confirm their kills instead of shooting someone into crit and running off"
or B) "have the victims' heads comically explode into a shower of gibs and grey matter and encourage them to go watch Netflix for an hour"
A... or B?

**Pull request merged**